pythran/config.py

```diff
@@ -21,20 +26,67 @@ def init_cfg(sys_file, platform_file, user_file):
     user_config_dir = os.environ.get('XDG_CONFIG_HOME', '~')
     user_config_path = os.path.expanduser(
         os.path.join(user_config_dir, user_file))
+    return {"sys": sys_config_path, "platform": platform_config_path, "user": user_config_path}
+
 
-    cfgp = configparser.SafeConfigParser()
+def init_cfg(sys_file, platform_file, user_file):
+    paths = get_paths_cfg(sys_file, platform_file, user_file)
+    sys_config_path, platform_config_path, user_config_path = paths.values()
+
+    cfgp = ConfigParser()
     for required in (sys_config_path, platform_config_path):
         cfgp.readfp(open(required))
     cfgp.read([user_config_path])
 
-    for obsolete_section in ('user', 'sys'):
-        if cfgp.has_section(obsolete_section):
-            logger.warn("Your pythranrc has an obsolete `%s' section",
-                        obsolete_section)
-
     return cfgp
 
 
+def lint_cfg(cfgp, **paths):
+    if len(paths) == 0:
+        paths = get_paths_cfg()
+
+    # Use configuration from sys and platform as "reference"
+    cfgp_ref = ConfigParser()
+    cfgp_ref.read([paths["sys"], paths["platform"]])
+
+    # Check if pythran configuration files exists
+    for loc, path in paths.items():
+        exists = os.path.exists(path)
+
+        msg = "{} file {} \n\t ↳ {}".format(
```

Comment:
↳ is a non-ascii character, and it makes the python2 build crash. Can you use another formating.
I would replace

```
INFO     sys file exists 
	 ↳ /home/avmo/src/pythran/pythran/pythran.cfg
```

by

```
INFO     sys file exists:  /home/avmo/src/pythran/pythran/pythran.cfg
```

sorry for the eye candies ;-)

Reply:
I had hoped `from __future__ import unicode_literals` this would manage it... :disappointed: . Let's do it after 2020 then :wink: